vcd/resource_vcd_any_access_control_test.go

```diff
@@ -0,0 +1,263 @@
+//go:build access_control || vdc || catalog || vapp || ALL || functional
+// +build access_control vdc catalog vapp ALL functional
+
+package vcd
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/vmware/go-vcloud-director/v2/govcd"
+)
+
+// TestAccVcdAnyAccessControlGroups checks that vcd_org_vdc_access_control, vcd_vapp_access_control,
+// and vcd_catalog_access_control resources are capable of correctly handling group IDs correctly.
```

Comment:
```suggestion
// and vcd_catalog_access_control resources are capable of handling group IDs correctly.
```
double use of "correctly"